frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -53,10 +84,28 @@ const Navbar = ({children}) => {
-              <p key={index} className="text-lg" onClick={() => setShowMenu(false)}>
+              <p
+                key={index}
+                className="text-lg hover:text-gray-500"
+                onClick={() => setShowMenu(false)}
+              >
                 <Link to={link.path}>{link.name}</Link>
               </p>
             ))}
+            {isLoggedIn ? (
+                <p className="text-lg hover:text-gray-500">
+                  <button className="cursor-pointer" onClick={handleSignOut}>Sign Out</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)